test/test_deploy.py

```diff
@@ -3,21 +3,41 @@
 from typing import List
 import pytest
 
-from starkware.starknet.business_logic.transaction.objects import InternalDeploy
-from starkware.starknet.core.os.contract_address.contract_address import (
-    calculate_contract_address,
-)
 from starkware.starknet.services.api.contract_class import ContractClass
 from starkware.starknet.services.api.gateway.transaction import Deploy
-from starkware.starknet.services.api.feeder_gateway.response_objects import (
-    TransactionStatus,
-)
 from starknet_devnet.constants import SUPPORTED_TX_VERSION
 
-from starknet_devnet.devnet_config import parse_args, DevnetConfig
-from starknet_devnet.starknet_wrapper import StarknetWrapper
-from .shared import CONTRACT_PATH, GENESIS_BLOCK_NUMBER
-from .util import assert_hex_equal
+from .util import (
+    assert_contract_class,
+    assert_negative_block_input,
+    assert_transaction_not_received,
+    assert_transaction_receipt_not_received,
+    assert_block,
+    assert_contract_code,
+    assert_equal,
+    assert_receipt,
+    assert_salty_deploy,
+    assert_storage,
+    assert_transaction,
+    assert_tx_status,
+    deploy,
+    get_class_by_hash,
+    get_class_hash_at,
+    get_full_contract,
+    get_block,
+)
+
+from .shared import (
+    BALANCE_KEY,
+    CONTRACT_PATH,
+    EVENTS_CONTRACT_PATH,
+    EXPECTED_SALTY_DEPLOY_ADDRESS,
+    EXPECTED_SALTY_DEPLOY_HASH,
+    EXPECTED_SALTY_DEPLOY_HASH_LITE_MODE,
+    EXPECTED_SALTY_DEPLOY_BLOCK_HASH_LITE_MODE,
+    GENESIS_BLOCK_NUMBER,
+    NONEXISTENT_TX_HASH,
+)
 
 
 def get_contract_class():
```

Comment:
Can you remove this function? It's only used by a function that could also be removed since it's not used (`get_deploy_transaction`)